armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -414,6 +416,32 @@ cd /opt/shift/tools/bbbfancontrol
 cp bbbfancontrol /usr/local/sbin/
 cp bbbfancontrol.service /etc/systemd/system/
 
+## base-middleware
+mkdir -p $GOPATH/src/github.com/shiftdevices && cd "$_"
+#cd $GOPATH/src/github.com/shiftdevices
+git clone https://github.com/shiftdevices/base-middleware
+cd base-middleware
+make native
+cp base-middleware /usr/local/sbin/
+cp base-middleware.service /etc/systemd/system/
```

Comment:
I guess copying the `base-middleware.service` is redundant when also writing it from within this script?

Reply:
Yup, thanks :)